build.assets/Dockerfile

```diff
@@ -83,6 +84,22 @@ RUN (groupadd ci --gid=$GID -o && useradd ci --uid=$UID --gid=$GID --create-home
 RUN (curl -L https://github.com/coreos/etcd/releases/download/v3.3.9/etcd-v3.3.9-linux-${BUILDARCH}.tar.gz | tar -xz && \
      cp etcd-v3.3.9-linux-${BUILDARCH}/etcd* /bin/)
 
+# Install libcbor.
+RUN git clone --depth=1 https://github.com/PJK/libcbor.git -b v0.9.0 && \
+  cd libcbor && \
+  cmake -DCMAKE_BUILD_TYPE=Release -DCBOR_CUSTOM_ALLOC=ON . && \
+  make && \
+  make install
+
+# Install libfido2.
+# Depends on libcbor, libssl-dev, zlib and libudev-dev.
+RUN git clone --depth=1 https://github.com/Yubico/libfido2.git -b 1.10.0 && \
+  cd libfido2 && \
+  cmake -DCMAKE_BUILD_TYPE=Release . && \
+  make && \
+  make install
+ENV LD_LIBRARY_PATH="/usr/local/lib:$LD_LIBRARY_PATH"
```

Comment:
Isn't `/usr/local/lib` in the library path by default? This will also likely leave `LD_LIBRARY_PATH` with a trailing colon, which might also include the current directory inadvertently.

Reply:
> Isn't /usr/local/lib in the library path by default

That was my expectation too, but I couldn't get it to work without setting the variable.

> This will also likely leave LD_LIBRARY_PATH with a trailing colon, which might also include the current directory inadvertently

Wouldn't have to be `:.`? I can drop the `:$LD_LIBRARY_PATH` if we think it's best.

Reply:
Maybe run `ldconfig` after the `make install`?

Reply:
Removed trailing `:$LD_LIBRARY_PATH`.

Reply:
`ldconfig` seems better, thanks. Updating PR soon.